packages/ui/src/screens/proposal_details/utils.tsx

```diff
@@ -12,7 +12,7 @@ export const getProposalType = (proposalType: string) => {
     type = 'softwareUpgradeProposal';
   }
 
-  if (proposalType === '/cosmos.upgrade.v1beta1.CommunityPoolSpendProposal') {
+  if (proposalType === '/cosmos.distribution.v1beta1.CommunityPoolSpendProposal') {
```

Comment:
just to confirm that this is an intended change, not merge by my mistake, right?

Reply:
@teamchong hey yes it is, the old path was wrong